logger/src/config.rs

```diff
@@ -0,0 +1,99 @@
+//! Module containing logic related to spawning a logger from the
+//! configuration, as well as run-time reloading of the log-level.
+
+use iroha_config::{
+    derive::Configurable,
+    runtime_upgrades::{ReloadFailure, ReloadHandle, Reloadable},
+};
+use serde::{Deserialize, Serialize};
+use tracing::Subscriber;
+use tracing_subscriber::{filter::LevelFilter, reload::Handle};
+
+const DEFAULT_MAX_LOG_LEVEL: LevelEnv = LevelEnv::DEBUG;
+
+use super::{debug, Level};
+
+/// Log level for reading from environment and (de)serializing
+#[allow(clippy::upper_case_acronyms)]
+#[derive(Debug, Deserialize, Serialize, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+pub enum LevelEnv {
```

Comment:
I am not sure about naming, right now we don't only read from env, but we also have post endpoint and config.json

Reply:
So why name it `LevelEnv`...

Reply:
I didn't want to change too many things at once. Since it's now in a completely separate module I can just call it `Level` and use the appropriate namespace when ambiguous.